### PR TITLE
Update monthly call page with link to forum

### PR DESCRIPTION
**Pull request by jancborchardt**
Please review @ei8fdb @evalica @dmichl :)

I now moved all notes from the monthly calls we had on Github over to the forum at https://discourse.opensourcedesign.net/c/meta/monthly-call – slowly we’re migrating. :D

### CODE REVIEW

**Review by evalica — APPROVED** [review]

File: osd-recurring-online-meeting.md [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
-time:  7:00 PM to ... (Berlin Time)
+time:  19:00-20:00 (Berlin Time)
 status: upcoming
 permalink: /open-source-design-monthly-online-meeting/
 ---
 
 Join us for our monthly get together where we talk about all things Open Source Design. Everyone is welcome.
 
-To see what we talked about in previous meetings, look at:
+To see what we talked about in previous meetings, check out the ["monthly call" category on the forum](https://discourse.opensourcedesign.net/c/meta/monthly-call).
```
Comment (evalica) [inline osd-recurring-online-meeting.md]: The only thing is that the category is called "Monthly call", with a capitalized M :) but besides this, looking good. Thanks Jan.

Comment (ei8fdb) [reply osd-recurring-online-meeting.md]: Suggestion:

``To get notifications of this meeting and other events, you can subscribe to our Open Source Design events calendar: [Open Source Design calendar](https://cloud.nextcloud.com/index.php/apps/calendar/p/MIFAFLFJADIVX63I/Open-Source-Design)``

Comment (ei8fdb) [reply osd-recurring-online-meeting.md]: Thoughts?

Comment (jancborchardt) [reply osd-recurring-online-meeting.md]: Added :)

Comment (ei8fdb) [reply osd-recurring-online-meeting.md]: Looks good to me. @evalica ?

**Pull request merged**